src/platform/ESP32/OTAImageProcessorImpl.cpp

```diff
@@ -237,7 +261,7 @@ CHIP_ERROR OTAImageProcessorImpl::SetBlock(ByteSpan & block)
         {
             ReleaseBlock();
         }
-        uint8_t * mBlock_ptr = static_cast<uint8_t *>(chip::Platform::MemoryAlloc(block.size()));
```

Comment:
Why are these changes done?

Reply:
This code is inside namespace chip, so using `chip::` is not required so, cleaning up.